frontend/src/static/js/v_authorship.js

```diff
@@ -129,6 +146,11 @@ window.vAuthorship = {
               ...acc, [key]: unsortedFilesInfoObj[key]
           }), {});
     },
+
+    getBlankLineCount(fileType) {
```

Comment:
This method name makes me think I'm just getting the blank line of the entire analysis counts as an `int`, but it returns a `string` instead with info on non-blank line count as well. Perhaps rename to `getFileBlankLineInfo`?

Reply:
ok, have renamed the method